charmsen-hometown.md

```diff
@@ -0,0 +1,7 @@
+# Tucson, AZ
+
+## Places to eat
```

Comment:
@charmsen. I am travelling to Tuscon next week. I like Stake What place do you  recommend?